pysmiles/read_smiles.py

```diff
@@ -210,8 +243,16 @@ def read_smiles(smiles, explicit_hydrogen=False, zero_order_bonds=True,
                 raise KeyError(f'Node {node} ({format_atom(mol, node)}) has'
                                f' non-standard valence: ...{debug_smiles}...')
 
+    # post-processing of E/Z isomerism
+    _annotate_ez_isomers(mol, ez_isomer_pairs)
+
+    # post-processing of chiral atoms
+    # potentially we need all hydrogen in place
+    _mark_chiral_atoms(mol)
```

Comment:
can we hide these behind an API flag? For cgsmiles of course this results in an error

Reply:
on the other hand I would still have to grep the ezisomer pairs. On second though I will filter them out and then do them in the end